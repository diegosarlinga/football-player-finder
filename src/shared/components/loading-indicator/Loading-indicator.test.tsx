import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import LoadingIndicator from './Loading-indicator';

describe('LoadingIndicator component tests', () => {
  let wrapper: ShallowWrapper;

  describe('when rendered', () => {
    beforeEach(() => {
      wrapper = shallow(<LoadingIndicator/>);
    });
  
    it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });

  });
});
