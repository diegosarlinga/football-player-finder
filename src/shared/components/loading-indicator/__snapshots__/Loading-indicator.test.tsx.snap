// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator component tests when rendered should match snapshot 1`] = `
<div
  className="Loading-indicator-overlay d-flex justify-content-center align-items-center"
>
  <div
    className="Loading-indicator spinner-border text-primary"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
